### PR TITLE
Revise readme

### PR DESCRIPTION
**Pull request by jffng**
#### Changes proposed in this Pull Request:

This PR updates the main readme a bit to clarify the purpose of this repo for folks inside and outside of automattic.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,39 +1,44 @@
...
+The development repository for themes maintained by the [Automattic Theme Team](https://themeshaper.com/about/). 
 
-These themes are maintained by the [Automattic Theme Team](https://themeshaper.com/about/).
+## What's Here
 
-## Submitting issues
+We're committed to imagining and creating the future of themes. Our latest themes are [block themes](https://developer.wordpress.org/block-editor/how-to-guides/themes/block-theme-overview/) — they depend on the latest features available in the [Gutenberg plugin](https://wordpress.org/plugins/gutenberg/). Read about our latest themes built to support full-site editing [here](https://wordpress.com/blog/2022/03/09/customize-your-entire-site-with-new-block-themes/).
 
-Before submitting your issue, make sure it has not been discussed earlier. You can search for existing tickets [here](https://github.com/Automattic/themes/search).
+<img width="1399" alt="Screen Shot 2022-03-21 at 1 31 48 PM" src="https://user-images.githubusercontent.com/5375500/159330814-8f8883f3-1cd4-44a6-a404-05c8f800ac93.png">
```
Comment (jffng) [inline README.md]:
```suggestion
<img width="1399" alt="A gallery of block themes with a variety of styles" src="https://user-images.githubusercontent.com/5375500/159330814-8f8883f3-1cd4-44a6-a404-05c8f800ac93.png">
```

**Comment by MaggieCabrera** [comment]
I like your new additions! But I re-read the whole readme file and I think the bit that explains the sandbox tools is referencing old commands.

**Comment by jffng** [comment]
> I think the bit that explains the sandbox tools is referencing old commands.

True, I will update those.

**Comment by pbking** [comment]
I've removed references to out-of-date utilities and replaced with the modern version.  There are a number of other utilities, but I didn't mention them.  Ready for a [look-see](https://github.com/Automattic/themes/blob/update/readme/README.md).

**Review by mikachan — APPROVED** [review]
I think this is a great update to our readme 🚢

**Comment by jffng** [comment]
Thank you everyone!

**Pull request merged**